packages/experiments-realm/review.gts

```diff
@@ -0,0 +1,198 @@
+import {
+  Component,
+  field,
+  contains,
+} from 'https://cardstack.com/base/card-api';
+import { formatDatetime } from './blog-app';
+import { BlogPost } from './blog-post';
+import { RatingsSummary } from './ratings-summary';
+
+// @ts-expect-error using own template
+export class Review extends BlogPost {
```

Comment:
Extending a card def and changing one of its templates gives error about "incorrectly extending BlogPost class"  "types of property 'isolated' are incompatible"